package.json

```diff
@@ -21,7 +21,7 @@
     "test:watch:integration": "concurrently --raw npm:dev 'npm:test:open:integration'",
     "test:es-check": "npm run test:es-check:es5 && npm run test:es-check:es2015:module",
     "test:es-check:es5": "es-check es5 'dist/auth0-spa-js.production.js'",
-    "test:es-check:es2015:module": "es-check es2015 --module 'dist/auth0-spa-js.production.esm.js'",
+    "test:es-check:es2015:module": "es-check es2015 'dist/auth0-spa-js.production.esm.js' --module ",
```

Comment:
There was a bug upgrading `es-check` to >= 6.1, and this change fixes it. [Reported here](https://github.com/yowainwright/es-check/issues/139)